ApplicationCode/Commands/RicElasticPropertiesImportTools.cpp

```diff
@@ -85,7 +85,8 @@ void RicElasticPropertiesImportTools::importElasticPropertiesFromFile( const QSt
         }
     }
 
-    RimElasticProperties* rimElasticProperties = new RimElasticProperties;
+    RimElasticProperties* rimElasticProperties = fractureModelTemplate->elasticProperties();
+    if ( !rimElasticProperties ) rimElasticProperties = new RimElasticProperties;
```

Comment:
Potential memory leak, who owns rimElasticProperties  after new operation?

Reply:
@magnesj It is always assigned to the FractureModelTemplate in line: 
https://github.com/OPM/ResInsight/blob/2453407ccfcb5ace69a1c0ef009b879ca31bdec2/ApplicationCode/Commands/RicElasticPropertiesImportTools.cpp#L150

The assignment is not strictly necessary when it did already exist, but figured that the code was clean if we always assign.